test/test-helpers/mockConfig.js

```diff
@@ -0,0 +1,36 @@
+const faker = require('faker');
```

Comment:
Nit: Can you use kabob-case with the filename?

Reply:
done!